src/main/java/teammates/common/datatransfer/StudentAttributesFactory.java

```diff
@@ -118,52 +115,57 @@ private int locateColumnIndexes(String headerRow) throws EnrollException {
         hasName = false;
         hasEmail = false;
         hasComment = false;
-        
+
         String[] columns = splitLineIntoColumns(headerRow);
-        
+
         for (int curPos = 0; curPos < columns.length; curPos++) {
             String str = columns[curPos].trim().toLowerCase();
-            
-            if(StringHelper.isAnyMatching(str, FieldValidator.REGEX_COLUMN_SECTION)){
+
+            if (StringHelper.isAnyMatching(str,
+                    FieldValidator.REGEX_COLUMN_SECTION)) {
```

Comment:
Either not have the line break or indent 2nd line to align with the `str` parameter. When breaking a group of parameters to multiple lines, use indentation to show which parameters belong together.